portifolio.html

```diff
@@ -139,5 +139,36 @@ <h3>Multi-Post Stories</h3>
         </div>
     </div>
 </section>
+<section class="contact-form">
+    <h1>
+        CONTACT US
+    </h1>
+    <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+    </p>
+    <form action="#">
```

Comment:
Your form action path is undefined. We actually want the form to be directed to a [formspree service](https://formspree.io/html).[formspree service](https://formspree.io/html). Therefore kindly improve upon othis aspect by ensuring that:
- [ ] `action` value is similar to `https://formspree.io/f/{form_id}`
- [ ] `method` value equals `POST`